docs/modules/ROOT/examples/queue.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: hazelcast.com/v1alpha1
+kind: Queue
+metadata:
+  name: queue-sample
+spec:
+  hazelcastResourceName: hazelcast
+  maxSize: 10
+  emptyQueueTTLSeconds: 100
```

Comment:
AFAIK, we are showing all exposed field with default values in the example yaml.